File: class09/python/AP0911-BoasPraticasNomes.py
# BoasPraticasNomes — Aula 09
# Demonstra boas práticas de nomenclatura de variáveis

# Nomes significativos revelam a intenção
salario_bruto = 5000.00
desconto_inss = salario_bruto * 0.11
desconto_ir = salario_bruto * 0.075
salario_liquido = salario_bruto - desconto_inss - desconto_ir
print(f"Salário Bruto: R$ {salario_bruto:.2f}")
print(f"Desconto INSS: R$ {desconto_inss:.2f}")
print(f"Desconto IR:   R$ {desconto_ir:.2f}")
print(f"Salário Líquido: R$ {salario_liquido:.2f}")

File: class09/java/AP0911-BoasPraticasNomes.java
public class BoasPraticasNomes {
    public static void main(String[] args) {
        double salarioBruto = 5000.00;
        double descontoInss = salarioBruto * 0.11;
        double descontoIr = salarioBruto * 0.075;
        double salarioLiquido = salarioBruto - descontoInss - descontoIr;
        System.out.printf("Salário Bruto: R$ %.2f%n", salarioBruto);
        System.out.printf("Desconto INSS: R$ %.2f%n", descontoInss);
        System.out.printf("Desconto IR:   R$ %.2f%n", descontoIr);
        System.out.printf("Salário Líquido: R$ %.2f%n", salarioLiquido);
    }
}